beaker-testmode_switcher.gemspec

```diff
@@ -17,6 +17,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "beaker"
+  if RUBY_VERSION <= "2.1.6"
+    spec.add_dependency "beaker", "2.42.0"
+  elsif RUBY_VERSION > "2.1.6"
+    spec.add_dependency "beaker"
+  end
```

Comment:
gemspecs can't "really" do ruby-version-dependent dependencies. Putting it here will cause the gem to contain a versioned dependency iff it is built on a ruby 2.1.6 or older. That might be of lesser concern to us internally, but I'd loath to leave that landmine around.

Perhaps repeating the dependency in the Gemfile is enough? If that leads to duplicate gem errors, I'd rather see it fixed there (by not using the gemspec information), than this risk here.